source/beamline/BioXAS/BioXASMainBeamline.h

```diff
@@ -57,12 +58,30 @@ class BioXASMainBeamline : public AMBeamline
 
 	/// Destructor.
 	virtual ~BioXASMainBeamline();
-
+    /// Returns true if all beamline components are connected, false otherwise.
```

Comment:
That is what mine are set to as well.  Do you have "Insert spaces instead of tabs" enabled?